### PR TITLE
Completed Day 2 - Please Review.

### PR DESCRIPTION
**Pull request by bharatreddypothireddy**

### CODE REVIEW

File: QUsingStack.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+    public static void main(String[] args) {
+        QUsingStack q = new QUsingStack();
+        q.push(2);
+        q.push(1);
+        System.out.println(q.peek());
+        System.out.println(q.pop());
+        System.out.println(q.empty());
+    }
+}
```
Comment (super30admin) [inline QUsingStack.java]: Thats a very nice solution Bharat.
Suggestion:
    You can implement this solution even without using all the length variables(using same approach).

File: MyHashSet.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        MyHashSet hashSet = new MyHashSet();
+        hashSet.add(1);
+        hashSet.add(2);
+        System.out.println(hashSet.contains(1));    // returns true
+        System.out.println(hashSet.contains(3));    // returns false (not found)
+        hashSet.add(2);
+        System.out.println(hashSet.contains(2));    // returns true
+        hashSet.remove(2);
+        System.out.println(hashSet.contains(2));    // returns false (already removed)
+    }
```
Comment (super30admin) [inline MyHashSet.java]: Very Well Done !!!

File: QUsingStack.java (hunk 1 above)
Comment (bharatreddypothireddy) [reply QUsingStack.java]: Implemented without using length variables.

File: MyHashSet.java (hunk 2 above)
Comment (bharatreddypothireddy) [reply MyHashSet.java]: Implemented using a 2d boolean array

**Comment by super30admin** [comment]
Thats a great job Bharat!!

**Pull request closed without merging**